lib/net/http.rb

```diff
@@ -1090,7 +1090,7 @@ def write_timeout=(sec)
     end
 
     # Returns the continue timeout value.
-    # See Net::HTTP.continue_timeout=.
+    # See Net::HTTP.continue_timeout=. 
```

Comment:
Please remove the trailing space.

```suggestion
    # See Net::HTTP.continue_timeout=.
```

Reply:
Fixed.